Servers/UI/OJS.Servers.Ui/ClientApp/src/layout/header/PageHeader.tsx

```diff
@@ -15,12 +17,13 @@ import logo from './softuni-logo-horizontal.svg';
 import styles from './PageHeader.module.scss';
 
 const PageHeader = () => {
-    const { state: { user } } = useAuth();
-
     const { actions: { toggleVisibility } } = useSearch();
-
+    const { data, isSuccess } = useGetUserinfoQuery(null);
```

Comment:
Rename these variables so its easier to read in the next lines of code

Reply:
Fixed